Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -273,9 +296,17 @@ namespace AZ::Render
             const float parentBoneScale = CalculateBoneScale(instance, skeleton->GetNode(parentIndex));
             const float cylinderSize = boneLength - boneScale - parentBoneScale;
 
+            if (selectedJoints.find(jointIndex) != selectedJoints.end())
+            {
+                renderColor = SelectedColor;
+            }
+            else
+            {
+                renderColor = skeletonColor;
+            }
             // Render the bone cylinder, the cylinder will be directed towards the node's parent and must fit between the spheres
-            auxGeom->DrawCylinder(centerWorldPos, boneDirection, boneScale, cylinderSize, skeletonColor);
-            auxGeom->DrawSphere(nodeWorldPos, boneScale, skeletonColor);
+            auxGeom->DrawCylinder(centerWorldPos, boneDirection, boneScale, cylinderSize, renderColor);
+            auxGeom->DrawSphere(nodeWorldPos, boneScale, renderColor);
```

Comment:
drawShaded=false? I think generally that looks nicer for debug rendering, especially with the current lighting setup, or are you planning to adjust and play around with that?

Reply:
![image](https://user-images.githubusercontent.com/69218254/143247380-20b7db76-3317-430c-b5ac-f23c087216f1.png)
this is what it is looked like changing from drawstyle::shade to solid. I don't think that looks better actually? Or you mean something else?

Reply:
Hm yeah doesn't look great indeed